deploy/cryostat.yml

```diff
@@ -0,0 +1,337 @@
+apiVersion: v1
+kind: Template
+metadata:
+  name: cryostat
+objects:
+- apiVersion: v1
+  kind: ServiceAccount
+  metadata:
+    name: cryostat
+    labels:
+      helm.sh/chart: cryostat-0.3.1
+      app.kubernetes.io/name: cryostat
+      app.kubernetes.io/instance: cryostat
+      app.kubernetes.io/version: "k.redhat"
+      app.kubernetes.io/managed-by: Helm
+- apiVersion: v1
+  kind: Secret
+  metadata:
+    name: cryostat-jmx-credentials-db
+  type: Opaque
+  data:
+    CRYOSTAT_JMX_CREDENTIALS_DB_PASSWORD: "a0JWQ2VqeEFuemI4MExCTmtldEJzY3pudHkyZVRzRTY="
```

Comment:
This Secret/its content probably shouldn't be committed

Reply:
good point, not sure how this exactly works here and/or can we just simply remove this?  can it function without DB? @ebaron

Reply:
Perhaps this secret can be added using Vault in app-interface. We could do something similar to enable basic authentication as well.

Reply:
Sorry totally overlooked this. Probably should come from vault yes. In terms of generation, it could be generated also by the application itself (not sure if that would be a good approach)

Reply:
I've removed the generated Secret. The secret can now be obtained from Vault.